capstone/src/main/webapp/WEB-INF/web.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
+<web-app xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://java.sun.com/xml/ns/javaee"
+  xmlns:web="http://java.sun.com/xml/ns/javaee/web-app_2_5.xsd"
+  xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_2_5.xsd"
+  version="2.5">
+  <servlet>
+<display-name>Remote API Servlet</display-name>
+<servlet-name>RemoteApiServlet</servlet-name>
+<servlet-class>com.google.apphosting.utils.remoteapi.RemoteApiServlet</servlet-class>
+<load-on-startup>1</load-on-startup>
+</servlet>
+<servlet-mapping>
+<servlet-name>RemoteApiServlet</servlet-name>
+<url-pattern>/remote_api</url-pattern>
+</servlet-mapping>
+
+<servlet>
+<display-name>Restaurants Servlet</display-name>
+<servlet-name>RestaurantsServlet</servlet-name>
+<servlet-class>com.google.step.servlets.RestaurantsServlet</servlet-class>
+<load-on-startup>1</load-on-startup>
+</servlet>
+<servlet-mapping>
+<servlet-name>RestaurantsServlet</servlet-name>
+<url-pattern>/restaurants</url-pattern>
+</servlet-mapping>
+
+<servlet>
+<display-name>Login Servlet</display-name>
+<servlet-name>LoginServlet</servlet-name>
+<servlet-class>com.google.step.servlets.LoginServlet</servlet-class>
+<load-on-startup>1</load-on-startup>
+</servlet>
+<servlet-mapping>
+<servlet-name>LoginServlet</servlet-name>
+<url-pattern>/login</url-pattern>
+</servlet-mapping>
+
+<servlet>
+<display-name>Page View Servlet</display-name>
+<servlet-name>PageViewServlet</servlet-name>
+<servlet-class>com.google.step.servlets.PageViewServlet</servlet-class>
+<load-on-startup>1</load-on-startup>
+</servlet>
+<servlet-mapping>
+<servlet-name>PageViewServlet</servlet-name>
+<url-pattern>/page-view</url-pattern>
+</servlet-mapping>
+
+<servlet>
+<display-name>Restaurant Servlet</display-name>
+<servlet-name>RestaurantServlet</servlet-name>
+<servlet-class>com.google.step.servlets.RestaurantServlet</servlet-class>
+<load-on-startup>1</load-on-startup>
+</servlet>
+<servlet-mapping>
+<servlet-name>RestaurantServlet</servlet-name>
+<url-pattern>/restaurant</url-pattern>
+</servlet-mapping>
+</web-app>
```

Comment:
Can all of this be tabified for readability purposes?

Reply:
Sorry about that, added correct formatting